include/alpaka/rand/RandStl.hpp

```diff
@@ -69,15 +90,91 @@ namespace alpaka
                     std::mt19937 m_State;
                 };
 
+                //#############################################################################
+                //! "Tiny" state mersenne twister implementation
+                //!
+                //! repository: github.com/MersenneTwister-Lab/TinyMT
+                //!
+                //! @license 3-clause BSD
```

Comment:
clang complains: `RandStl.hpp:98:21: error: unknown command tag name [-Werror,-Wdocumentation-unknown-command]`
We have to be compatible with the strict subset of documentation that clang understands...